src/Form/Type/TodoType.php

```diff
@@ -0,0 +1,33 @@
+<?php
+namespace App\Form\Type;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\SubmitType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\Extension\Core\Type\HiddenType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use App\Entity\Todo;
+
+class TodoType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('task', TextType::class)
+            ->add('status', HiddenType::class, [
+              'data' => 'Incomplete'
+            ])
+            ->add('save', SubmitType::class, [
+              'label' => 'Create Task'
+            ])
+        ;
+    }
+
+    public function configureOptions(OptionsResolver $resolver)
+    {
+        $resolver->setDefaults([
+            'data_class' => Todo::class,
+        ]);
```

Comment:
This explicit defining of class type should allow for embedded forms later on without issues.